crates/divan_compat/src/compat/mod.rs

```diff
@@ -12,30 +12,52 @@ pub mod __private {
 
 mod bench;
 mod entry;
+mod util;
+
+pub use bench::*;
 
 pub fn main() {
+    // Outlined steps of original divan::main and their equivalent in codspeed instrumented mode
     // 1. Get registered entries
     // TODO: Manage bench groups
     let bench_entries = &entry::BENCH_ENTRIES;
 
     // 2. Build an execution tree
-    // TODO:
+    // No need, we do not manage detailed tree printing like original divan, and we extract
+    // codspeed URI from entry metadata directly.
 
-    // 3. Filter the tree then sort it (drop sort?)
-    // TODO:
+    // 3. Filtering
+    // No need, filtering is managed at a higher level in the `cargo-codspeed` wrapper.
```

Comment:
cargo-codspeed will handle pasckage filtering but we might want a bench level filtering as well here. And by bench level, I mean below the bench target but really the benchmark within the target.
This is probably not handled in criterion either, so it's okay not to handle it for now, but we should mention it's not supported right now (and maybe add a warning when the flags are passed?)